workspaces/bulk-import/plugins/bulk-import/src/types/types.ts

```diff
@@ -98,6 +98,19 @@ export enum RepositoryStatus {
   'CODEOWNERS_FILE_NOT_FOUND_IN_REPO' = 'CODEOWNERS_FILE_NOT_FOUND_IN_REPO',
 }
 
+export enum AddedRepositoryColumnNameEnum {
+  repoName = 'repository.name', // because of property name
+  repositoryName = 'repository.name',
+  organizationUrl = 'repository.organization',
+  repoUrl = 'repository.url',
+  catalogInfoYamlLastUpdated = 'lastUpdate',
+  catalogInfoYamlStatus = 'status',
+}
```

Comment:
Maybe it would be good to unify this and not repeat values. If `repoName` is needed, you can remove `repositoryName` and instead always use `repoName`.

```suggestion
export enum AddedRepositoryColumnNameEnum {
  repoName = 'repository.name',
  organizationUrl = 'repository.organization',
  repoUrl = 'repository.url',
  catalogInfoYamlLastUpdated = 'lastUpdate',
  catalogInfoYamlStatus = 'status',
}
```